app/classifier/components/TaskNavButtons/components/TalkLink/TalkLink.jsx

```diff
@@ -34,7 +34,7 @@ export const StyledTalkLink = styled(Link)`
 export const StyledDisabledTalkPlaceholder = styled.span`
   ${commonStyles}
   opacity: 0.5;
-  pointer-events: 'none';
+  pointer-events: none;
```

Comment:
Nice solution to disabling links. Does it disable them from the keyboard too? If not, consider adding `tabindex="-1"` when the link is disabled.